build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -892,6 +886,11 @@ message OutputFile {
 
   // True if file is executable, false otherwise.
   bool is_executable = 4;
+
+  // The contents of the file if inlining was requested. The server SHOULD NOT inline
+  // file contents unless requested by the client in the
+  // [GetActionResultRequest][build.bazel.remote.execution.v2.GetActionResultRequest]
+  bytes contents = 5;
```

Comment:
I know that usually we don't reuse removed tags, but this is a rare case of the exact same field being removed and then ressurrected. I think this justifies using bytes contents = 3, removing the "reserved".

Reply:
Meh. I'd prefer to stick to the proto guidelines if you don't mind.